packages/store/src/secure-store/index.ts

```diff
@@ -12,6 +15,23 @@ function getKeytar(log: Logger): typeof Keytar | undefined {
         log.warn(errorString(err));
         log.warn(`Could not "require('keytar')". Trying VSCode's copy`);
         let vscode;
+
+        const keytarGlobVscode = join(
+            homedir(),
+            '.vscode/extensions/sapse.sap-ux-application-modeler-extension-**/node_modules/keytar/package.json'
+        );
+        const keytarGlobVscodeInsiders = join(
+            homedir(),
+            '.vscode-insiders/extensions/sapse.sap-ux-application-modeler-extension-**/node_modules/keytar/package.json'
+        );
+        const files = globSync([keytarGlobVscode, keytarGlobVscodeInsiders]);
+        console.log('files: \n' + JSON.stringify(files));
```

Comment:
remove `console.log` or replace with `log.info`

Reply:
Thank you, applied ✅